ui/src/kapacitor/constants/index.js

```diff
@@ -118,4 +165,5 @@ export const ALERT_NODES_ACCESSORS = {
         ['alerta()']
       )
       .join('.'),
+  pushover: rule => _.get(rule, 'alertNodes[0].args'),
 }
```

Comment:
Do you want a default return value from this get if `alernNodes[0].args` is undefined?